src/collections/blog/2023/2023-08-07-layer5-badge-program/index.mdx

```diff
@@ -0,0 +1,194 @@
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/meshery-operator/meshery-operator.svg" />
+  <span>Meshery Operator</span> <p>This badge awarded is awared to the community members who make consistent and impactful contributions to Meshery Operator of the Meshery project in recognition and appreciation of their efforts.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/docker-extension/docker-extension.svg" /> 
+  <span>Docker Extension</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the Docker Extension of meshery project in recognition and appreciation of their efforts.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/meshery-docs/meshery-docs.svg" /> <span>Meshery Docs</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the <a href="https://docs.meshery.io">Meshery Docs</a> in recognition and appreciation of their efforts.</p>
+</div>
+</div>
+
+<h3>Earn your first badge today</h3>
+
+Are you ready to start earning badges? Join the [Layer5](http://layer5.io/) community platform and start making your contributions! By earning badges, you don't just highlight your accomplishments - you contribute to the growth and success of Layer5. Let's celebrate your milestones, your dedication, and your passion for open source together. By obtaining a Layer5 Badge, you have the opportunity to inspire others on their journey and shine.
+
+Keep an eye out for more updates, challenges, and badge releases. Together, let's shape the future of cloud native together! Connect with us on [Twitter](https://twitter.com/layer5), [LinkedIn](https://www.linkedin.com/company/layer5), and [GitHub](https://github.com/meshery/meshery) to share your badge-earning journey and be a part of the vibrant Layer5 community. Star our [repository](https://github.com/meshery/meshery) and help us cross 3,000.
+
+Happy contributing and badge-earning! 🚀🛡️
+
+### FAQ's
+
+**Is there a limit to the number of badges that I can earn?**
+There is no set limit to the number of badges you can earn within the Layer5 Recognition Program. Your achievements and contributions determine the badges you receive, showcasing your dedication and skills within the community. 
+
+**How do I share my badges on other social profiles?**
+To share your badges on other social profiles, utilize the provided embedded code and direct share functionality. Access your badges from the Layer5 Cloud profile badges section, and then use it to seamlessly post your achievements on social media platforms. This enables you to showcase your Layer5 contributions and milestones to a wider audience. 
+
```

Comment:
```suggestion
 Embed your Layer5 badges in your GitHub profile
@leecalcote Since I can't update the PR, here is the example
```

![image](https://github.com/layer5io/layer5/assets/92504666/e139be86-2d5b-426a-a841-a22bec16fdfb)